developers/arbitrum-deploy.md

```diff
@@ -8,13 +8,42 @@ We will go over installation of Arbitrum Nitro and deploying an instance on an
 Ubuntu AMD machine. This section covers all necessary dependencies needed to be
 installed.
 
+## Compatibility matrix
+
+| Component | Version | Details |
+|-----------|---------|---------|
+| Nitro | [v2.3.1](https://github.com/celestiaorg/nitro/releases/tag/v2.3.1) | Includes the replay binary for the WASM root `0xa17295e3918d39e4026503302f1d627608101a52d4341647ebf7ad18edbe31a3`. [Docs on overall changes](https://github.com/celestiaorg/nitro/blob/celestia-v2.3.1/docs/celestia/docs.md). |
+| Contracts | [v1.2.1-celestia](https://github.com/celestiaorg/nitro-contracts/releases/tag/v1.2.1-celestia) | Integrates Blobstream X functionality into nitro-contracts v1.2.1 |
+| Orbit SDK | [v0.8.2 Orbit SDK for Celestia DA](https://github.com/celestiaorg/arbitrum-orbit-sdk/releases/tag/v0.8.2) | This is not compatible with Orbit SDK v0.8.2 or with the latest changes to nitro-contracts for the Atlas upgrade. The Orbit SDK itself is in Alpha. |
+| celestia-node | [v0.13.1](https://github.com/celestiaorg/celestia-node/releases/tag/v0.13.1) | This integration has only been tested with celestia-node 0.13.1 and only works with said version, and with future versions after that. Under the hood, the Nitro node uses [this commit](https://github.com/celestiaorg/celestia-openrpc/commit/64f04840aa97d4deb821b654b1fb59167d242bd1) of celestia-openrpc. |
```

Comment:
🅱️

Reply:
moving to appendix. feels out of place at top